frontend/components/common/Layout/Footer.tsx

```diff
@@ -286,32 +291,41 @@ function NewsletterForm({
                <FooterNewsletterEmailLabel>
                   Enter your email
                </FooterNewsletterEmailLabel>
-               <FooterNewsletterEmailInput
-                  ref={inputRef}
-                  disabled={subscription.status !== SubscriptionStatus.Idle}
-                  placeholder={emailPlaceholder}
-               />
+               <HStack>
+                  <FooterNewsletterEmailInput
+                     ref={inputRef}
+                     disabled={subscription.status !== SubscriptionStatus.Idle}
+                     placeholder={emailPlaceholder}
+                     hidden={isSubscribed}
+                  />
+                  {isSubscribed && (
+                     <Text
+                        align="center"
+                        position={'absolute'}
+                        top="2"
+                        left="0"
+                        right="5"
+                     >
+                        <Icon as={RiCheckFill} boxSize="5" mb="-5px" mr="6px" />
+                        Subscribed!
+                     </Text>
+                  )}
+                  <Button
+                     width="50%"
+                     type="submit"
+                     data-testid="submit-button"
```

Comment:
Since this is global on the page, let's make this something like footer-newsletter-subscribe-button

Reply:
Resolved 9c4d74a80bc3c6f2cd8bb5e6ce7f6f02a2f8cd85